napari/_tests/test_multiple_viewers.py

```diff
@@ -15,3 +18,4 @@ def test_multi_viewers_dont_clash(qapp):
 
     v1.close()
     v2.close()
+    qtbot.wait(50)
```

Comment:
If there's a specific explanation about why we need the arbitrary wait at the end (e.g. the 50ms comes from the throttled timeout duration), then add it. If not, then I'll just close my eyes and ignore this for now 🙈

Reply:
I try to follow this https://github.com/napari/napari/blob/main/napari/utils/_testsupport.py#L173
but decide to stop this debug and wait on @tlambert03 response to if he knows why we use directly `Viewer` here.